Cargo.toml

```diff
@@ -26,3 +26,4 @@ sha256 = "1.5.0"
 syn = "1.0"
 tempfile = "3.3.0"
 wasmparser = "0.119.0"
+pocket-ic = { path = "/home/mwe/dup/ic/packages/pocket-ic" }
```

Comment:
TODO: Use next PocketIC version. The current one does not support large instruction limits.